packages/core/src/classes.ts

```diff
@@ -271,6 +271,7 @@ export interface Space extends Doc {
   private: boolean
   members: Arr<Ref<Account>>
   archived: boolean
+  presenter?: AnyAttribute
```

Comment:
Why we need to extend base Space class?

Reply:
Please use some of view.mixin.* if you need presenters.

Reply:
Corrected it